docs/scripts/postMigration.ts

```diff
@@ -289,6 +298,10 @@ function run() {
 /components/* /material-ui/react-:splat 301
 /:lang/components/* /:lang/material-ui/react-:splat 301
 
+/api/unstable-trap-focus/ /base/api/trap-focus/ 301
+/zh/api/unstable-trap-focus/ /zh/base/api/trap-focus/ 301
+/pt/api/unstable-trap-focus/ /pt/base/api/trap-focus/ 301
```

Comment:
It doesn't seem to be migration-related. It's already needed in production. Proof: https://mui.com/api/unstable-trap-focus/

Reply:
Yes, I am aware of it. My plan is to bundle this fix into the next big release, that's why I put it in the postMigration so that I don't have to remember what needs to be fixed manually. And yes, It will be 404 for the next few days but I am not sure how we can deploy only this fix without the rest of the work before it.

To not delay this fix, I will merge it. Feel free to share your thought then we can discuss them further.